geode-web-api/src/main/java/org/apache/geode/rest/internal/web/controllers/AbstractBaseController.java

```diff
@@ -983,4 +974,16 @@ protected QueryService getQueryService() {
     targetedMembers.add(cache.getDistributedSystem().getDistributedMember());
     return targetedMembers;
   }
+
+  protected String[] parseKeys(HttpServletRequest request, String region) {
+    String uri = request.getRequestURI();
```

Comment:
Did `request.getAttribute(HandlerMapping.PATH_WITHIN_HANDLER_MAPPING_ATTRIBUTE)` not work?

Reply:
No it did not. It gave the entire uri. This was in the context of the RestAccessControllerTest

Reply:
Another thing that did not do what was expected was "request.getContextPath()". It returned an empty string. I found an example that expected it to be the first part of the URL up to the start of {region}.